releasenotes/notes/dualstack-profile.yaml

```
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: feature
+area: installation
+issue: []
+
+releaseNotes:
+- |
+  **Added** profile to install Istio in dual-stack experimental mode.
```

Do we want to call out the name as well?
```suggestion
  **Added** dual stack profile to install Istio in dual-stack experimental mode.
```